.github/workflows/qemu-emulator-build.yaml

```
@@ -174,13 +225,11 @@ jobs:
 
       - name: Stop emulator
         if: always()
-        run: |
-          EMULATOR_ARCH=${{ matrix.arch }} \
-            docker/local-emulator/qemu/run-emulator.sh stop
+        run: node packages/stack-cli/dist/index.js emulator stop
 
       - name: Print serial log on failure
         if: failure()
-        run: tail -100 docker/local-emulator/qemu/run/vm/serial.log 2>/dev/null || true
+        run: tail -100 $HOME/.stack/emulator/run/vm/serial.log 2>/dev/null || true
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:11: Double quote to prevent globbing and word splitting

(shellcheck)